_posts/en/newsletters/2021-04-14-newsletter.md

```diff
@@ -0,0 +1,225 @@
+---
+title: 'Bitcoin Optech Newsletter #144'
+permalink: /en/newsletters/2021/04/14/
+name: 2021-04-14-newsletter
+slug: 2021-04-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes recent progress on code to activate
+taproot and contains our regular sections with descriptions of a recent
+Bitcoin Core PR Review Club meeting and notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Taproot activation discussion:** since our last update on discussion
+  about [activation methods][topic soft fork activation] for the
+  [taproot][topic taproot] soft fork in [Newsletter #139][news139
+  activation], the
+  Speedy Trial proposal became the focus of attention among those
+  interested in activation.  PRs were opened for two variants of it:
+  [PR#21377][Bitcoin Core #21377], using a variation on [BIP9][], and
+  [PR#21392][Bitcoin Core #21392], using a modification that became part
+  of [BIP8][].  The main technical difference between these PRs is how
+  their start and stop points were specified.  PR#21377 uses Median Time
+  Past ([MTP][]); PR#21392 uses the height of the current block.
+
+    MTP is normally roughly consistent between Bitcoin's main network
+    (mainnet) and its various testnet works, such as testnet, the default
```

Comment:
"testnet works" almost works, but I gather it was intended to be "test networks"